app/models/like.rb

```diff
@@ -0,0 +1,11 @@
+class Like < ApplicationRecord
+  belongs_to :post
```

Comment:
- [ ] Just as I mentioned in my earlier comment, from the ERD Diagram you can see that a like also belongs to both a user and a post but you have only established the association to a post and not to a user. So kindly fix that :+1: